src/ui/atoms/icon/Icon.tsx

```diff
@@ -44,18 +44,25 @@ export type IconProps = {
    * Custom className to add some additional style.
    */
   readonly className?: string
+  /**
+   * Defines if icon theme style is inverted.
+   */
+  readonly invert?: boolean
```

Comment:
I would have included the color notion into the property's name, because `invert` can cause confusion: what is inverted? The icon itself? The color? Its spatial representation?

Something like: 

```
 /**
  * If `true`, the color is inverted according to the current theme.
  */
  readonly invertColor?: boolean
```

Reply:
I agree